test/functional/test_framework/mempool_util.py

```diff
@@ -19,6 +19,7 @@
     MiniWallet,
 )
 
+MAX_ORPHANS = 100
```

Comment:
Let's keep the same name in the functional test for uniformity and easy reference?
https://github.com/bitcoin/bitcoin/blob/master/src/net_processing.h#L27

Reply:
Sounds good to me! Updated in [ad38457](https://github.com/bitcoin/bitcoin/pull/31040/commits/ad384572cad35a248ab0322e9f8855bbed024f0b)